services/client/src/pages/events/reducers/events.js

```diff
@@ -1,6 +1,7 @@
-import { updateEventsList } from "../utils";
+import { updateEventsList, parsedDomain } from "../utils";
 
 const defaultState = {
+  location: parsedDomain.subdomain,
```

Comment:
We should inject the subdomain from the page container before `fetchData`.

```suggestion
  location: "",
```
```